.github/workflows/vale-lint.yml

```diff
@@ -0,0 +1,25 @@
+name: Linting
+on:
+  pull_request:
+    branches: [latest]
+
+jobs:
+  prose:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+
+    - name: Vale
+      uses: errata-ai/vale-action@v1.4.0
+      with:
+        # Optional
+        styles: |
+          https://github.com/errata-ai/Microsoft/releases/latest/download/Microsoft.zip
+          https://github.com/errata-ai/write-good/releases/latest/download/write-good.zip
```

Comment:
can we please add the Google style, instead of the MS one? Write-good is fine, though 👍